Headlines/src/Services/ContentViewsService.swift

```diff
@@ -32,34 +31,32 @@ class ContentViewsService: HTTPService {
                 let err = NSError(domain: "ContentViewsService",
                                   code: 1,
                                   userInfo: errUserInfo)
-                fail?(err)
+                handler?(.failure(err))
                 return
             }
 
-            let successBlock: (_ result: Data?, _ response: URLResponse?) -> Void = {(data, response) in
-                DispatchQueue.main.async(execute: {
-                    success?(response)
-                })
-            }
-
-            let failBlock: (_ error: NSError) -> Void = { (e) in
-                DispatchQueue.main.async(execute: {
-                    fail?(e as NSError)
-                })
-            }
-
             let params = [
                 "news_id": newsIdentifier,
                 "user_id": userId.recordName,
                 "user_source": "iOS",
                 "context_from": context.rawValue
                 ]
 
-            _ = self.request(method: .POST,
-                             path: "content-views/",
-                             params: params,
-                             success: successBlock,
-                             fail: failBlock)
+            let httpHandler: ((Result <Data?, Error>) -> Void) = { result in
+                switch result {
+                case .success(_):
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/cafeb7b8636cbafd0216eaa2489bb108aa52959c/Headlines/src/Services/ContentViewsService.swift#L47"></span>Arguments can be omitted when matching enums with associated types if they are not used.<br />
<code>empty_enum_arguments</code> <code>ContentViewsService.swift:47</code></td>
    </tr>
  </tbody>
</table>